pandas/tseries/plotting.py

```diff
@@ -1,313 +1,3 @@
-    if isinstance(data.index, DatetimeIndex):
-        base = frequencies.get_freq(freq)
-        x = data.index
-        if (base <= frequencies.FreqGroup.FR_DAY):
-            return x[:1].is_normalized
-        return Period(x[0], freq).to_timestamp(tz=x.tz) == x[0]
-    return True
-
-
-def _get_index_freq(data):
-    freq = getattr(data.index, 'freq', None)
-    if freq is None:
-        freq = getattr(data.index, 'inferred_freq', None)
-        if freq == 'B':
-            weekdays = np.unique(data.index.dayofweek)
-            if (5 in weekdays) or (6 in weekdays):
-                freq = None
-    return freq
-
-
-def _maybe_convert_index(ax, data):
-    # tsplot converts automatically, but don't want to convert index
-    # over and over for DataFrames
-    if isinstance(data.index, DatetimeIndex):
-        freq = getattr(data.index, 'freq', None)
-
-        if freq is None:
-            freq = getattr(data.index, 'inferred_freq', None)
-        if isinstance(freq, DateOffset):
-            freq = freq.rule_code
-
-        if freq is None:
-            freq = _get_ax_freq(ax)
-
-        if freq is None:
-            raise ValueError('Could not get frequency alias for plotting')
-
-        freq = frequencies.get_base_alias(freq)
-        freq = frequencies.get_period_alias(freq)
-
-        data = data.to_period(freq=freq)
-    return data
-
-
-# Patch methods for subplot. Only format_dateaxis is currently used.
-# Do we need the rest for convenience?
-
-
-def format_dateaxis(subplot, freq):
-    """
-    Pretty-formats the date axis (x-axis).
-
-    Major and minor ticks are automatically set for the frequency of the
-    current underlying series.  As the dynamic mode is activated by
-    default, changing the limits of the x axis will intelligently change
-    the positions of the ticks.
-    """
-    majlocator = TimeSeries_DateLocator(freq, dynamic_mode=True,
-                                        minor_locator=False,
-                                        plot_obj=subplot)
-    minlocator = TimeSeries_DateLocator(freq, dynamic_mode=True,
-                                        minor_locator=True,
-                                        plot_obj=subplot)
-    subplot.xaxis.set_major_locator(majlocator)
-    subplot.xaxis.set_minor_locator(minlocator)
-
-    majformatter = TimeSeries_DateFormatter(freq, dynamic_mode=True,
-                                            minor_locator=False,
-                                            plot_obj=subplot)
-    minformatter = TimeSeries_DateFormatter(freq, dynamic_mode=True,
-                                            minor_locator=True,
-                                            plot_obj=subplot)
-    subplot.xaxis.set_major_formatter(majformatter)
-    subplot.xaxis.set_minor_formatter(minformatter)
-
-    # x and y coord info
-    subplot.format_coord = lambda t, y: (
-        "t = {0}  y = {1:8f}".format(Period(ordinal=int(t), freq=freq), y))
-
-    pylab.draw_if_interactive()
```

Comment:
or I see, you may have left all of the pandas/tseries namespace as is (and just moved things), ok then.